.circleci/config.yml

```diff
@@ -329,13 +329,13 @@ commands:
             cd packages/functional-tests/tests
             TEST_FILES=$(circleci tests glob "./**/*.spec.ts")
             cd ..
-            echo $TEST_FILES | circleci tests run --command="xargs yarn playwright test --project=<< parameters.project >> $GREP" --verbose --split-by=timings --timings-type=classname
+            echo $TEST_FILES | circleci tests run \
+              --command="xargs yarn playwright test --project=<< parameters.project >> $GREP" \
+              --verbose \
+              --split-by=timings \
+              --timings-type=classname
           environment:
             NODE_OPTIONS: --dns-result-order=ipv4first
-            JEST_JUNIT_OUTPUT_DIR: ./artifacts/tests
-            JEST_JUNIT_ADD_FILE_ATTRIBUTE: true
```

Comment:
Not sure why these are here, they have no impact for Playwright.

Reply:
I'm not sure either...